core/domain/prod_validation_jobs_one_off.py

```diff
@@ -136,6 +136,28 @@ def _get_external_id_relationships(cls, item):
     def _get_validation_functions(cls):
         return []
 
+    @classmethod
+    def cleanup(cls, item):
```

Comment:
Actually, I think we should write a fully independent, targeted one-off job for the cleanup, and not tie it to the validator. This is because, in general, different "issues" will have different mitigation methods.

So I suggest that we just have a standalone one-off job in user_jobs_one_off.py to handle this. If there's any association with the validation job class, it should actually be with the specific type of problem discovered rather than the entire class itself.

Reply:
I agree different issues will have different migration methods, but code to fetch external models can be shared?
I've moved this clean up logic out of the Validator class, PTAL and let me know what you think

Reply:
Ah -- to clarify, what I meant was that the separate fields are actually separate "issues" IMO. So if the exploration_ids field contains stray IDs, then in general that would be a different issue from the creator_ids and general_feedback_thread_ids fields containing stray IDs and would, in general, be handled differently.

Hence the idea of having one focused job per "issue".

Reply:
Thanks for the clarification, changed the test to specifically remove offending exploration ids.